nxdrive/data/i18n/i18n-fr.json

```diff
@@ -116,7 +116,7 @@
     "DRIVE_ROOT_RECREATE": "Recréer",
     "DONT_ASK_AGAIN": "Ne plus afficher à l'avenir",
     "DOWNGRADE_TO": "Mettre à jour vers %1",
-    "DUPLICATE_BEHAVIOR": "Files duplicates management",
+    "DUPLICATE_BEHAVIOR": "Gestion des doublons",
```

Comment:
We should specify the "Files", right? Somethhng like "Gestion des fichiers doublons" (maybe horrible, but we need to be on the same page to prevent users misunderstandings.